addons/http_routing/views/http_routing_template.xml

```diff
@@ -122,7 +122,9 @@
                                 <div class="col-12">
                                     <p class="text-center pt24">Maybe you were looking for one of these <b>popular pages?</b></p>
```

Comment:
As `.pt24` is on a paragraph, it's not editable.
Move this class to the column (which should be `.col-lg-12,` not `.col-12`).

Reply:
What about removing this `pt24` and setting the padding of the `hr` above (div in L120) to `p-5` instead?

Reply:
Yes! And the `.s_hr` snippet should be used here. 
Like in the second slide of the carousel snippet:
[https://github.com/odoo/odoo/blob/def8eb4b51a80587f2b4a17da82ceb3abb7f017c/addons/website/views/snippets/s_carousel.xml#L35](https://github.com/odoo/odoo/blob/def8eb4b51a80587f2b4a17da82ceb3abb7f017c/addons/website/views/snippets/s_carousel.xml#L35)